ref.yml

```diff
@@ -59,6 +59,13 @@ fields:
       type: bool
       used:
         - README.md
+  - nix:
+      type: bool
+      description: >
+        Set to true to generate a default.nix file and to test it.
+      used:
+        - README.md
+        - .travis.yml
```

Comment:
is a `- 'nix-action.yml'` missing here?

Reply:
No, it's used to decide whether to generate the file but not *in* the file.

Reply:
OK! sorry for the noise :)